packages/integration-sdk-cli/src/neo4j/neo4jGraphStore.ts

```diff
@@ -142,17 +154,20 @@ export class Neo4jGraphStore {
       );
 
       const buildCommand = `
-      MERGE (start {_key: $startEntityKey, _integrationInstanceID: $integrationInstanceID})
-      MERGE (end {_key: $endEntityKey, _integrationInstanceID: $integrationInstanceID})
+      MERGE (start${startEntityType} {_key: $startEntityKey, _integrationInstanceID: $integrationInstanceID})
```

Comment:
Can you teach me here? This look different

Reply:
So if you are able to add a label to the node in a MERGE statement it lets Neo4j use indexes to find it more quickly (MERGE effectively does a MATCH statement to find it and then either updates or creates accordingly).  The ${startEntityType} and ${endEntityType} are two such potential labels, depending on if the collected data has type information collected (which so far looks like an optional thing to me, but I'm still investigating a bit further).  We have to do string interpolation because (last I checked) Neo4j doesn't allow labels to be parameterized like the other items we have in this Cypher statement.